BuyBitcoin/settings.py

```diff
@@ -141,14 +148,11 @@
 USE_TZ = True
 
 
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.9/howto/static-files/
-
+STATIC_ROOT = os.path.join(PROJECT_ROOT, 'staticfiles')
```

Comment:
Change back to static

Reply:
We have static and static files (two do different things - staticfiles go in the source folder, buybitcoin in this case, while static lives in root)

Reply:
Then why did the Heroku deploy fail?